schema/deploy/roles/presence-absence-processor.sql

```diff
@@ -0,0 +1,31 @@
+-- Deploy seattleflu/schema:roles/presence-absence-processor to pg
+
+begin;
+
+create role presence_absence_processor;
+
+grant connect on database :"DBNAME" to presence_absence_processor;
+
+grant usage
+   on schema receiving, warehouse
+   to presence_absence_processor;
+
+grant select
+   on receiving.presence_absence
+   to presence_absence_processor;
+
+grant update (processing_log)
+   on receiving.presence_absence
+   to presence_absence_processor;
+
```

Comment:
@tsibley : Jover and I were curious to know why this role does not receive `UPDATE` privileges on `receiving.presence_absence` despite the fact we're granting permissions here to `processing_log`. 

When we run `select * from information_schema.table_privileges where grantee='presence-absence-processor;` we see that there is only the `SELECT` `privilege_type` for `receiving.presence_absence`.

We modeled this role after `enrollment-processor` and see the same `privilege_type` for `receiving.enrollment` there, too.

Reply:
You can learn more about `information_schema` here: https://www.postgresql.org/docs/10/information-schema.html